ql/currency.hpp

```diff
@@ -116,6 +116,10 @@ namespace QuantLib {
     /*! \relates Currency */
     bool operator!=(const Currency&,
                     const Currency&);
+ 
+    /*! \relates Currency */
+    bool operator<(const Currency&,
```

Comment:
Why would you sort currencies?  I'm not sure to say (in the interface) that one currency is greater than another.